src/dash-table/components/Export/index.tsx

```diff
@@ -34,7 +36,7 @@ export default React.memo((props: IExportButtonProps) => {
 
     return (<div>
         {!isFormatSupported ? null : (
-            <button className='export' onClick={handleExport}>Export</button>
+            <button className={export_className} onClick={handleExport}>{export_text}</button>
```

Comment:
@Marc-Andre-Rivet is the `export` class important to us internally? Should we instead be keeping it and appending the user-provided value?